### PR TITLE
Text Refresh for Operation Move and Resize

### PR DESCRIPTION
**Pull request by byancey**
-Update the operation text and some of the dialog text for both move and resize.

-Added Key-up event to trigger validate for the NumericUpDown margin boxes in the move operation so that values are validated as entered, rather than only when focus is lost.  This addresses an issue where you can enter a value that pushes the model out of bounds, but it's not actually validated until focus when the "Move" button is actually pressed.  With this change, "Move" will be disabled before it can be pressed.

### CODE REVIEW

**Comment by byancey** [comment]
Added commit with Flip, Morph, Rotate, and Solidify

File: UVtools.GUI/Controls/Tools/CtrlToolFlip.Designer.cs [hunk 1]
```diff
@@ -48,9 +48,11 @@ private void InitializeComponent()
             this.cbMakeCopy.AutoSize = true;
             this.cbMakeCopy.Location = new System.Drawing.Point(240, 8);
             this.cbMakeCopy.Name = "cbMakeCopy";
-            this.cbMakeCopy.Size = new System.Drawing.Size(104, 24);
+            this.cbMakeCopy.Size = new System.Drawing.Size(120, 24);
             this.cbMakeCopy.TabIndex = 22;
-            this.cbMakeCopy.Text = "Make copy";
+            this.cbMakeCopy.Text = "Blend Layers";
+            this.toolTip.SetToolTip(this.cbMakeCopy, "If checked, rather than simply flipping the layer, a copy of each layer will be f" +
```
Comment (sn4k3) [inline UVtools.GUI/Controls/Tools/CtrlToolFlip.Designer.cs:54]: Append string unnecessary, merge them

File: UVtools.GUI/Controls/Tools/CtrlToolResize.Designer.cs [hunk 2]
```diff
@@ -46,9 +46,12 @@ private void InitializeComponent()
             this.cbFade.AutoSize = true;
             this.cbFade.Location = new System.Drawing.Point(8, 41);
             this.cbFade.Name = "cbFade";
-            this.cbFade.Size = new System.Drawing.Size(170, 24);
+            this.cbFade.Size = new System.Drawing.Size(283, 24);
             this.cbFade.TabIndex = 26;
-            this.cbFade.Text = "Fade towards 100%";
+            this.cbFade.Text = "Increase or decrease towards 100%";
+            this.toolTip.SetToolTip(this.cbFade, "If checked, resize will gradually adjust the scale factor from the percentage spe" +
```
Comment (sn4k3) [inline UVtools.GUI/Controls/Tools/CtrlToolResize.Designer.cs:52]: Another concat, is designer marking this?

File: UVtools.GUI/Controls/Tools/CtrlToolRotate.Designer.cs [hunk 3]
```diff
@@ -91,6 +93,7 @@ private void InitializeComponent()
             this.Font = new System.Drawing.Font("Microsoft Sans Serif", 12F, System.Drawing.FontStyle.Regular, System.Drawing.GraphicsUnit.Point, ((byte)(0)));
             this.Name = "CtrlToolRotate";
             this.Size = new System.Drawing.Size(540, 45);
+            this.toolTip.SetToolTip(this, "  ");
```
Comment (sn4k3) [inline UVtools.GUI/Controls/Tools/CtrlToolRotate.Designer.cs:96]: Empty tooltip on ctrl window? Should be removed

**Comment by byancey** [comment]
I will push up a re-based pull-request with all operations to replace this pull request.

**Pull request closed without merging**